doc/services/logging/index.rst

```diff
@@ -846,6 +855,10 @@ Some of the platforms characterization for log message with two ``integer`` argu
 | x86_64        | 32       | 528                        | 1088      | 1440                        |
 +---------------+----------+----------------------------+-----------+-----------------------------+
 
+Logging using ARM Coresight STM
+*******************************
+
+For logging on NRF54H20 using ARM Coresight STM see :ref:`logging_cs_stm`.
```

Comment:
yes, but i wanted to have somewhere the link to that logging but i agree that maybe it would be better to have nrf54h20 specific page with all tips&tricks for that chip and have logging there.